Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs

```diff
@@ -48,7 +50,35 @@ public partial class SpatialMappingManager : Singleton<SpatialMappingManager>
         /// <summary>
         /// The current source of spatial mapping data.
         /// </summary>
-        public SpatialMappingSource Source { get; private set; }
+        public SpatialMappingSource Source
+        {
+            get { return source; }
+
+            private set
+            {
+                if (source != value)
+                {
+                    UpdateRendering(false);
+
+                    var oldSource = source;
+                    source = value;
+
+                    UpdateRendering(DrawVisualMeshes);
+
+                    var handlers = SourceChanged;
+                    if (handlers != null)
+                    {
+                        handlers(this, PropertyChangedEventArgsEx.Create(() => Source, oldSource, source));
```

Comment:
Nit: avoid use of lambdas per [HoloLens Unity Performance Recommendations](https://developer.microsoft.com/en-us/windows/holographic/performance_recommendations_for_unity)